OracleLinux/7/README.md

```diff
@@ -1,36 +1,89 @@
 # ol7-vagrant
-A vagrant box that provisions Oracle Linux automatically, using Vagrant, an Oracle Linux 7 box and a shell script.
+
+A vagrant project that provisions Oracle Linux automatically, using Vagrant, an Oracle Linux 7 box and a shell script.
 
 ## Prerequisites
+
 1. Install [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-2. Install [Vagrant](https://vagrantup.com/)
+1. Install [Vagrant](https://vagrantup.com/)
 
 ## Getting started
+
 1. Clone this repository `git clone https://github.com/oracle/vagrant-boxes`
-2. cd vagrant-boxes/OracleLinux/7
-3. Run `vagrant status` to check Vagrantfile status and possible plugin(s) required
-4. Run `vagrant up`
+1. cd vagrant-boxes/OracleLinux/7
+1. Run `vagrant status` to check Vagrantfile status and possible plugin(s) required
+1. Run `vagrant up`
    1. The first time you run this it will provision everything and may take a while. Ensure you have a good internet connection!
-   2. The Vagrant file allows for customization.
-5. SSH into the VM either by using `vagrant ssh`
+   1. The Vagrant file allows for customization.
+1. SSH into the VM either by using `vagrant ssh`
    If required, by Vagrantfile you can also setup ssh port forwarding.
-6. You can shut down the box via the usual `vagrant halt` and the start it up again via `vagrant up`.
+1. You can shut down the box via the usual `vagrant halt` and the start it up again via `vagrant up`.
 
 ## Optional plugins
 
 When installed, this Vagrantfile will make use of the following third party Vagrant plugins:
+
+- [vagrant-env](https://github.com/gosuri/vagrant-env): loads environment
+variables from .env files;
 - [vagrant-proxyconf](https://github.com/tmatilai/vagrant-proxyconf): set
 proxies in the guest VMs if you need to access the Internet through proxy. See
 plugin documentation for the configuration.
 - [vagrant-reload](https://github.com/aidanns/vagrant-reload): reload the VM
 during provisioning to activate the latest kernel.
 
-To intall Vagrant plugins run:
-```
+To install Vagrant plugins run:
+
+```shell
 vagrant plugin install <name>...
 ```
 
+## Extending the project
+
+This project can easily be extended by running additional scripts during provisioning and optionally expose guest ports on the host.
+
+Environment variables are used to pass the parameters to the Vagrantfile:
+
+- `EXTEND`: coma separated list of extensions.  
+   For each extension a script `<extension>.sh` is run. Scripts must be located in the `scripts` or `scripts.local` directory.
```

Comment:
"For each specified extension a script named `<extension>.sh` is automatically run during provisioning."